lib/private/Profile/ProfileManager.php

```diff
@@ -123,6 +129,24 @@ public function __construct(
 		$this->coordinator = $coordinator;
 	}
 
+	/**
+	 * If no user is passed as an argument return whether profile is enabled globally in `config.php`
+	 */
+	public function isProfileEnabled(?IUser $user = null): ?bool {
```

Comment:
Will always be a bool, no?

```suggestion
	public function isProfileEnabled(?IUser $user = null): bool {
```